src/components/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import cn from 'classnames';
+
+export const TodoList: React.FC<TodoListProps> = ({ todos, changeStatus }) => {
+  return (
+    <div>
+      {todos.map(({
+        completed, user, title, id,
+      }) => (
+        <button
+          type="button"
+          className= {cn('alert',
+            { 'alert-success': completed, 'alert-primary': !completed })}
```

Comment:
```suggestion
            { 
              'alert-success': completed,
              'alert-primary': !completed,
            })}
```
your code must be readable